NOTES-Perl.md

```diff
@@ -55,18 +55,22 @@
  ourselves to core Perl modules to keep the requirements down.  There
  are just a few exceptions:
 
-    Test::More      We require the minimum version to be 0.96, which
-                    appeared in Perl 5.13.4, because that version was
-                    the first to have all the features we're using.
-                    This module is required for testing only!  If you
-                    don't plan on running the tests, you don't need to
-                    bother with this one.
-
-    Text::Template  This module is not part of the core Perl modules.
-                    As a matter of fact, the core Perl modules do not
-                    include any templating module to date.
-                    This module is absolutely needed, configuration
-                    depends on it.
+  * `Test::More`
+
+  We require the minimum version to be 0.96, which
+  appeared in Perl 5.13.4, because that version was
+  the first to have all the features we're using.
+  This module is required for testing only!
+  If you don't plan on running the tests,
+  you don't need to bother with this one.
```

Comment:
I purposefully intendented this block of text to align with `Test::More`, so that it would be part of that list item.

Reply:
I see - done this way.